### DISCUSSION TITLE
Covid-19

### DISCUSSION THREAD

**Discussion by Covidkk** [Institut-Zdravotnych-Analyz/covid19-data#118] [Q&A]
Mohli by ste aj zverejniť koľko infikovaných detí denne pribúda.🙏🏻🙏🏻

**Comment by freedomlives** [comment]
Je to data na uroven vekove skupina (0 = 0-4 roky; 5 = 5-9 roky; atd):
https://github.com/Institut-Zdravotnych-Analyz/covid19-data/blob/main/PCR_Tests/OpenData_Slovakia_Covid_PositiveTests_AgeGroup_District.xlsx
